packages/firebase_ui_auth/pubspec.yaml

```diff
@@ -5,7 +5,7 @@ repository: https://github.com/firebase/flutterfire/tree/master/packages/firebas
 homepage: https://github.com/firebase/flutterfire/tree/master/packages/firebase_ui_auth
 
 environment:
-  sdk: '>=2.16.0 <3.0.0'
+  sdk: '>=2.19.0 <3.0.0'
```

Comment:
The rest of FlutterFire is still with >=2.16.0, I think we should keep everything aligned (so maybe up everything to 2.19 or keep everything to 2.16)